modules/acm_sku/src/ProductOptionsManager.php

```diff
@@ -130,7 +130,7 @@ public function loadProductOptionByOptionId($attribute_code, $option_id, $langco
    * {@inheritdoc}
    */
   public function createProductOptionWrapper($langcode, $option_id, $option_value, $attribute_id, $attribute_code, $weight) {
-    return $this->createProductOption();
+    return $this->createProductOption($langcode, $option_id, $option_value, $attribute_id, $attribute_code, $weight);
```

Comment:
This is a bug fix, yes?

Reply:
Yes, that was taken from the merge done from the earlier PR. If you want this moved out we can do it, but its pretty minor so I just kept it in.